packages/frontend-api/src/Resources/config/services.yaml

```diff
@@ -7,7 +7,6 @@ services:
             Overblog\DataLoader\DataLoaderInterface $filesBatchLoader: '@files_batch_loader'
             Overblog\DataLoader\DataLoaderInterface $firstImageBatchLoader: '@first_image_batch_loader'
             Overblog\DataLoader\DataLoaderInterface $categoriesBatchLoader: '@categories_batch_loader'
-            Overblog\DataLoader\DataLoaderInterface $productsSellableByIdsBatchLoader: '@products_sellable_by_ids_batch_loader'
```

Comment:
this throws the error for the reasons unknown to me

```
 A binding is configured for an argument of type "Overblog\DataLoader\DataLoaderInterface" named "$productsSellableByIdsBatchLoade
  r" under "_defaults" in file "/var/www/html/packages/frontend-api/src/DependencyInjection/../Resources/config/services.yaml", but
   no corresponding argument has been found. It may be unused and should be removed, or it may have a typo.
```

we may investigate further